velox/expression/tests/ExpressionFuzzer.cpp

```diff
@@ -245,14 +250,22 @@ bool useTypeName(
 }
 
 bool isSupportedSignature(const exec::FunctionSignature& signature) {
+  if (!FLAGS_velox_fuzzer_enable_custom_types) {
+    if (useTypeName(signature, "json") || useTypeName(signature, "opaque") ||
```

Comment:
If someone adds new custom type, this will stop working. Can we create an allow-list instead of a block-list?